controller/api/destination/watcher/prometheus.go

```diff
@@ -34,6 +34,68 @@ type (
 	}
 )
 
+var (
+	informer_lag_secs_buckets = []float64{
+		0.5,  // 500ms
+		1,    // 1s
+		2.5,  // 2.5s
+		5,    // 5s
+		10,   // 10s
+		25,   // 25s
+		50,   // 50s
+		100,  // 1m 40s
+		250,  // 4m 10s
+		1000, // 16m 40s
+	}
+	endpointsInformerLag = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "endpoints_informer_lag_secs",
+			Help:    "The amount of time between when an Endpoints resource is updated and when an informer observes it",
+			Buckets: informer_lag_secs_buckets,
+		},
+	)
+
+	endpointsliceInformerLag = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "endpointslice_informer_lag_secs",
```

Comment:
I've just noticed, I think it's better to use the plural form to have it aligned with the other metric `endpointslices_cache_size`
```suggestion
endpointslices_informer_lag_secs
```